src/diffusers/loaders.py

```diff
@@ -887,6 +887,9 @@ def load_lora_weights(self, pretrained_model_name_or_path_or_dict: Union[str, Di
         else:
             state_dict = pretrained_model_name_or_path_or_dict
 
+        if any("alpha" in k for k in state_dict.keys()):
+            state_dict = self._convert_kohya_lora_to_diffusers(state_dict)
```

Comment:
For my own understanding, this is the primary criterion that is deciding that this is a non-diffusers checkpoint, yeah?

Reply:
I think it's fairly decent starting point. We can tackle other criteria as more such conditions come up. WDYT?